students/gerry_gabrisch/lesson03/mailroom.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python3
+'''mailroom.py for lesson03, Gerry Gabrisch, started 4/2/2020'''
+
+import sys
+
+#a list of existing donations...
+donations = [('Eric Idle', [200.00, 500.00, 150.50]), 
+             ('John Cleese', [1000, 5000, 20]), 
+             ('Graham Chapman', [10000]), 
+             ('Terry Jones', [50, 600, 1000, 20]), 
+             ('Michael Palin', [100, 200]), 
+             ('Terry Gilliam',[3000])]
+
+def SendThanks(donations):
```

Comment:
function names should be snake_case per pep8 style guidelines -> `def send_thanks`